src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -40079,16 +40674,15 @@ static void F(object? x)
 }";
 
             var comp = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue(), targetFramework: TargetFramework.Mscorlib46);
+            // NullableWalker.VisitTupleExpression tracks tuple elements but not
+            // other (unexpected) fields of tuples (see y.F1.ToString()).
```

Comment:
Does this need an issue to track?

Reply:
I was assuming this is not a scenario we need to address, so I don't think we need an issue.

---
In reply to: [249094802](https://github.com/dotnet/roslyn/pull/32612#discussion_r249094802) [](ancestors = 249094802)

Reply:
I agree (I don't think we need to handle this scenario)

---
In reply to: [249098647](https://github.com/dotnet/roslyn/pull/32612#discussion_r249098647) [](ancestors = 249098647,249094802)